--- a/src/condition_variable.hpp
+++ b/src/condition_variable.hpp
@@ -83,7 +83,7 @@ namespace zmq
 #ifdef ZMQ_HAVE_WINDOWS
     std::mutex _m;
 #ifdef ZMQ_HAVE_CONDITION_VARIABLE
-    std::condition_variable_any _cv;
+    mutable std::condition_variable_any _cv;
 #endif
 #else
     pthread_mutex_t _m;
@@ -126,7 +126,7 @@ namespace zmq
 #ifdef ZMQ_HAVE_WINDOWS
     std::mutex _m;
 #ifdef ZMQ_HAVE_CONDITION_VARIABLE
-    std::condition_variable_any _cv;
+    mutable std::condition_variable_any _cv;
 #endif
 #else
     pthread_mutex_t _m;
